File: app/src/main/java/com/bmn/bookfinder/models/googlebooks/GBResponse.java
package com.bmn.bookfinder.models.googlebooks;

import com.bmn.bookfinder.models.ApiResponse;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class GBResponse extends ApiResponse implements Serializable {

    private final static long serialVersionUID = 2414488511190131242L;
    @SerializedName("kind")
    @Expose
    private String kind;
    @SerializedName("totalItems")
    @Expose
    private int totalItems;
    @SerializedName("items")
    @Expose
    private List<ResponseItem> items = null;

    public String getKind() {
        return kind;
    }

    public void setKind(String kind) {
        this.kind = kind;
    }

    public int getTotalItems() {
        return totalItems;
    }

    public void setTotalItems(int totalItems) {
        this.totalItems = totalItems;
    }

    public List<ResponseItem> getItems() {
        return items;
    }

    public void setItems(List<ResponseItem> items) {
        this.items = items;
    }

}

File: app/src/main/java/com/bmn/bookfinder/models/googlebooks/GBResponse.kt
package com.bmn.bookfinder.models.googlebooks

import com.bmn.bookfinder.models.ApiResponse
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class GBResponse : ApiResponse(), Serializable {
    @SerializedName("kind")
    @Expose
    var kind: String? = null

    @SerializedName("totalItems")
    @Expose
    var totalItems = 0

    @SerializedName("items")
    @Expose
    var items: List<ResponseItem> = emptyList()

    companion object {
        private const val serialVersionUID = 2414488511190131242L
    }
}